extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java

```diff
@@ -205,8 +211,16 @@ public void initialize() {
         }
 
         if (getUDN() != null) {
-            updateStatus(ThingStatus.ONLINE);
             onUpdate();
+            
+            if (getConfigAs(ZonePlayerConfiguration.class).notificationTimeout == null) {
+                Configuration c = editConfiguration();
+                c.put(ZonePlayerConfiguration.NOTIFICATION_TIMEOUT, DEFAULT_NOTIFICATION_TIMEOUT);
+                updateConfiguration(c);
+            }
+            
+            this.notificationTimeout = getConfigAs(ZonePlayerConfiguration.class).notificationTimeout;
```

Comment:
Please initiliaze the notification Timeout to the default here and don't use the NPE-safety check in the getter?